datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -344,7 +344,9 @@ public RangerCloudIdentitySyncStatus setRangerCloudIdentityMapping(String envCrn
     @Override
     @InternalOnly
     public RangerCloudIdentitySyncStatus getRangerCloudIdentitySyncStatus(String envCrn, long commandId) {
-        return rangerCloudIdentityService.getRangerCloudIdentitySyncStatus(envCrn, commandId);
+        return rangerCloudIdentityService.getRangerCloudIdentitySyncStatus(
+                envCrn,
+                Integer.valueOf(String.valueOf(commandId)));
```

Comment:
Why is this better than `(int) commandId`? Or do you perhaps depend on the `NumberFormatException` thrown by `Integer.valueOf()`?